kalite/topic_tools/__init__.py

```diff
@@ -460,8 +460,36 @@ def get_assessment_item_data(request, assessment_item_id=None):
 
     # TODO (rtibbles): Enable internationalization for the assessment_items.
 
+    try:
+        logging.info('Wrapping the Assessment Item with _()')
```

Comment:
I think this logging level might be too high. Could we change this and the one below to logging.debug or get rid of the logging statements altogether?